dashboard/pom.xml

```diff
@@ -78,7 +78,7 @@
         <artifactId>exec-maven-plugin</artifactId>
         <configuration>
           <skip>false</skip>
-          <exec>java</exec>
+          <executable>java</executable>
```

Comment:
https://www.mojohaus.org/exec-maven-plugin/examples/example-exec-for-java-programs.html

Reply:
so why does this work now? Or do we even need this goal?